test/remote/gateways/remote_mit_test.rb

```diff
@@ -0,0 +1,127 @@
+require 'test_helper'
+
+class RemoteMitTest < Test::Unit::TestCase
```

Comment:
When running the tests, I get three failing tests in this file:

```
Failure: test_successful_authorize_and_capture(RemoteMitTest)
Failure: test_successful_purchase(RemoteMitTest)
Failure: test_successful_refund(RemoteMitTest)
```

The failures all look similar:
```ruby
{
     "folio_cdp"=>"201397145",
     "auth"=>"",
     "response"=>"denied",
     "message"=>" -15- ",
     "id_comercio"=>"147",
     "reference"=>"TID|1615390643",
     "amount"=>"11.15",
     "time"=>"15:37:24 10:03:2021",
     "operation"=>"Authorize"
}
```

Any idea why these are not passing? I cannot find any documentation on this error code.